src/python/aqueduct_executor/operators/function_executor/execute.py

```diff
@@ -40,17 +40,32 @@ def _get_py_import_path(spec: FunctionSpec) -> str:
 
     if file_path.startswith("/"):
         file_path = file_path[1:]
-    return ".".join([OP_DIR, file_path.replace("/", ".")])
+    return file_path.replace("/", ".")
 
 
 def _import_invoke_method(spec: FunctionSpec) -> Callable[..., DataFrame]:
+    # `_import_invoke_method` imports the model object.
+    # it assumes the operator has been extracted to `<storage>/operators/<id>/op`
+    # and imports the route from the above path.
```

Comment:
what does "route" mean here?

nit: can we use the """ format for docstrings? I looks like its the standard and also looks way better on my IDE :D https://peps.python.org/pep-0257/